src/main/java/com/kuebiko/it/bibhusan/StringRemoval.java

```diff
@@ -0,0 +1,32 @@
+package com.kuebiko.it.bibhusan;
+
+/**
+ * Given two string arguments s1, s2
+ * return a string removing s2 from s1 ignoreCase
+ * return "N/A" if s2 is not found in s1
+ *
+ * eg. s1 = "HelloWorld", s2 = "World" => "Hello"
+ *  s1 = "JustDoIt", s2 = "do" => "JustIt"
+ *  s1 = "DoesntContain", s2 = "this" => "N/A"
+ */
+public class StringRemoval {
+
+  public String remove(String s1, String s2) {
+    String s1Copy=s1.toLowerCase();// created a copy of s1 and s2 to make it easier for comparing
```

Comment:
misleading variable name, its not actually a copy. Remember the principle of least surprise 😄